py3status/module.py

```diff
@@ -219,6 +252,9 @@ def run(self):
                         result['instance'] = self.module_inst
                         result['name'] = self.module_name
 
+                    # set universal module options in result
+                    result.update(self.module_options)
```

Comment:
So we just clobber anything set by the module?

I'm not necessarily opposed to this but good to be sure this is what we want.